app/resources/locale/en/en.po

```diff
@@ -9,30 +9,50 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.2.1\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.2.2\n"
 
-#: app/views/design.html:20
-msgid ": '}} {{'Unlock to edit block !"
+#: app/views/design.html:18
+msgid ""
+"<b>Read only</b><br>\n"
+"            Unlock to edit the block!"
 msgstr ""
+"<b>Read only</b><br>\n"
+"Unlock to edit the block!"
```

Comment:
msgid diverged from msgstr

Reply:
It's the other way around: msgstr is actually the *translation*, i.e. the one *diverging*. I removed the unnecessary spaces as they come from source code indentation and are redundant (HTML rendering collapses contiguous spaces). But I get your point here 👌